src/Hal/Metric/System/Packages/Composer/Packagist.php

```diff
@@ -36,7 +36,7 @@ public function get($package)
         }
         list($user, $name) = explode('/', $package);
         $uri = sprintf('https://packagist.org/packages/%s/%s.json', $user, $name);
-        $json = json_decode(@file_get_contents($uri));
+        $json = getURIContentAsJson($uri);
```

Comment:
I don't know if getting contents from external is something used in another part of the project, but maybe this function could be a method declared in another package to not be in a "messy" library of functions that does all stuff.

Reply:
I've moved it to the class as a private method. I thought it was going to be needed elsewhere, but it wasn't, so it could be there.